charm/docs/explanation/charm-architecture.md

```diff
@@ -0,0 +1,31 @@
+# NetBox charm architecture
+
+The NetBox charm initial version has been generated with the 
+help of the paas-app-charmer project. The paas-app-charmer project provides many
+of the functionalities needed by this charm like:
+- PostgreSQL integration
+- Django migrations
+- Ingress integration
+- COS (Prometheus metrics and Loki logs for gunicorn).
+
+For the static assets, gunicorn is used with the help of the [WhiteNoiseMiddleware](https://whitenoise.readthedocs.io/en/stable/index.html).
+
+There is only one container for each unit of NetBox, that runs the following
+services managed by Pebble:
+- django. Runs gunicorn.
+- cron. Cron service that runs management commands for housekeeping and syncdatasource.
```

Comment:
Is there a white space in `syncdatasource`?

Reply:
syncdatasource is the name of the django management command (see https://github.com/netbox-community/netbox/blob/develop/netbox/core/management/commands/syncdatasource.py)